sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/throughputControl/ThroughputControlTests.java

```diff
@@ -143,6 +145,69 @@ public void throughputGlobalControl(OperationType operationType) {
+        database.createContainerIfNotExists(controlContainer.getId(), "/groupId").block();
+
+        List<Pair<Integer, Boolean>> testCases = new ArrayList<>(
+            Arrays.asList(
+                Pair.of(6, true),
+                Pair.of(100, false)
+            )
+        );
+
+        UUID randomId = UUID.randomUUID();
+
+        for (Pair<Integer, Boolean> testCase : testCases) {
+            int targetThroughput = testCase.getLeft();
+            boolean shouldThrottleSecondRequest = testCase.getRight();
+
+            logger.info(
+                "TESTCASE - TargetThroughput: {}. ShouldThrottleSecondRequest: {}",
+                targetThroughput,
+                shouldThrottleSecondRequest);
+
+            // The create document in this test usually takes around 6.29RU, pick a RU here relatively close, so to test throttled scenario
+            ThroughputControlGroupConfig groupConfig =
+                new ThroughputControlGroupConfigBuilder()
+                    .groupName("group-" + randomId)
+                    .targetThroughput(targetThroughput)
+                    .build();
+
+            GlobalThroughputControlConfig globalControlConfig =
+                this.client.createGlobalThroughputControlConfigBuilder(this.database.getId(), controlContainerId)
+                           .setControlItemRenewInterval(Duration.ofSeconds(5))
+                           .setControlItemExpireInterval(Duration.ofSeconds(20))
+                           .build();
+            container.enableGlobalThroughputControlGroup(groupConfig, globalControlConfig);
+
+            CosmosItemRequestOptions requestOptions = new CosmosItemRequestOptions();
+            requestOptions.setContentResponseOnWriteEnabled(true);
+            requestOptions.setThroughputControlGroupName(groupConfig.getGroupName());
+
+            CosmosItemResponse<TestItem> createItemResponse = container.createItem(getDocumentDefinition(), requestOptions).block();
+            TestItem createdItem = createItemResponse.getItem();
+            this.validateRequestNotThrottled(
+                createItemResponse.getDiagnostics().toString(),
+                BridgeInternal.getContextClient(client).getConnectionPolicy().getConnectionMode());
+
+            // second request to same group. which will get throttled
+            CosmosDiagnostics cosmosDiagnostics = performDocumentOperation(this.container, operationType, createdItem, groupConfig.getGroupName());
+
+            if (shouldThrottleSecondRequest) {
+                this.validateRequestThrottled(
+                    cosmosDiagnostics.toString(),
+                    BridgeInternal.getContextClient(client).getConnectionPolicy().getConnectionMode());
+            } else {
+                this.validateRequestNotThrottled(
+                    cosmosDiagnostics.toString(),
+                    BridgeInternal.getContextClient(client).getConnectionPolicy().getConnectionMode());
+            }
+        }
+    }
+
```

Comment:
can we also add a test in enableSameGroupMultipleTimes:
after we change the threshold, we should also expect to see a new client item